ea_findcoords.m

```diff
@@ -1,5 +1,25 @@
 function coords=ea_findcoords(goodz,trajectory,trajvector,dist,correction,options)
-
+%
```

Comment:
Please provide a description of what this function is doing

Reply:
Seems deprecated, need input from @andreashorn